docs/modules/ROOT/pages/cops_lint.adoc

```diff
@@ -677,6 +677,41 @@ def initialize
 end
 ----
 
+== Lint/DuplicateBranch
+
+|===
+| Enabled by default | Safe | Supports autocorrection | VersionAdded | VersionChanged
+
+| Pending
+| Yes
+| No
+| 1.2
+| -
+|===
+
+This cop checks that there are no repeated bodies
+within `if/unless`, `case-when` and `rescue` constructs.
```

Comment:
I think we should add some examples with `case` and `rescue` as well.

Reply:
Added.